### DISCUSSION TITLE
Restoring PITR from same cluster

### DISCUSSION THREAD

**Discussion by isdinAghmir** [cloudnative-pg/cloudnative-pg#3244] [Q&A]
Hello I have a small question: 

Let's say I have a CNPG cluster named: cnpg-cluster01 , with scheduled backups and wal archiving. 

Something goes wrong and I drop all rows of a table. 

now I want to make a PITR from one of my scheduled backup, without an externalClusters. 

Is it possible to perform that ? Right now, applying my manifest results to nothing... except If I change the cluster name :( 

It would be great if we could be able to restore a cluster to a previous state, without having to change applications configuration that connects to the database.

**Comment by gbartolini** [comment]
If that happens, you should delete the original cluster. Then recreate a cluster with the same name, using the recovery bootstrap (either using a backup object or an external server). You should also add the annotation that skips WAL archive checking (`cnpg.io/skipEmptyWalArchiveCheck` set to `enabled`).

**Reply by isdinAghmir** [reply]
Thank you ! I m trying this in a few hours

**Reply by isdinAghmir** [reply]
works like a charm ! Thank you !
